runtime/builtin_types_test.go

```diff
@@ -23,6 +23,32 @@ import (
 	"testing"
 )
 
+func TestBuiltinDelAttr(t *testing.T) {
+	f := NewRootFrame()
+	delattr := mustNotRaise(Builtins.GetItemString(NewRootFrame(), "delattr"))
+	fooType := newTestClass("Foo", []*Type{ObjectType}, newStringDict(map[string]*Object{}))
+	fooForDelAttr := newObject(fooType)
+	SetAttr(f, fooForDelAttr, NewStr("bar"), None)
+	fun := wrapFuncForTest(func(f *Frame, args ...*Object) (*Object, *BaseException) {
+		_, raised := delattr.Call(f, args, nil)
+		if raised != nil {
+			return nil, raised
+		}
+		return GetAttr(f, args[0], toStrUnsafe(args[1]), nil)
```

Comment:
Yeah nativeInvoke() in native.go (which is used indirectly by wrapFuncForTest) [checks f.ExcInfo()](https://github.com/google/grumpy/blob/master/runtime/native.go#L562) when returning from invoking the function.

But you could instead just provide None instead of nil as the last arg to GetAttr() and then it'll return None instead of raising AttributeError.